### PR TITLE
Adding proper color to torque histogram widget

### LINKED ISSUE #135: Histogram on Time Series widgets should be orange and not green

### PR DESCRIPTION
**Pull request by xavijam**
After some thoughts, I feel the solution of adding a modifier class works ok, because you don't need to add colors in the JS views, only the class, and colors will be defined in the CSS. What do you think? 

screen shot 2016-02-11 at 18 23 28

CR: @javierarce 
cc @piensaenpixel @viddo

Fixes #135

### CODE REVIEW

File: src/widgets/histogram/chart.js [hunk 1]
```diff
@@ -5,11 +5,12 @@ var cdb = require('cartodb.js');
 var formatter = require('../../formatter');
 
 module.exports = cdb.core.View.extend({
-  defaults: {
+
+  options: {
```
Comment (xavijam) [inline src/widgets/histogram/chart.js]: Replaced by options, the natural way in the Backbone views.

File: themes/scss/widgets/themes/light.scss [hunk 2]
```diff
@@ -217,42 +219,40 @@ $chartMiniHighlight: rgba(#333, 1);
...
-
-  .CDB-Chart-bar.is-highlighted {
+  .CDB-Chart-bar--green.is-highlighted {
     fill: $darkHighlighted;
   }
-
+  .CDB-Chart-bar--orange {
+    fill: $highlightedAlternative;
+  }
+  .CDB-Chart-bar--orange.is-highlighted {
```
Comment (viddo) [inline themes/scss/widgets/themes/light.scss]: +1 to use modifier, although :-1: to use the color as the name… why not have something semantic, like `CDB-Chart--timeSeries`?

File: themes/scss/widgets/themes/light.scss [hunk 3]
```diff
@@ -217,42 +219,40 @@ $chartMiniHighlight: rgba(#333, 1);
...
   }
 
-
   //Charts
   .extent {
     stroke: $link;
   }
 
-  .CDB-Chart-bar {
+  .CDB-Chart-bar--green {
```
Comment (viddo) [inline themes/scss/widgets/themes/light.scss]: Isn't the green color the default color of a char bar? Is it necessary to have a modifier?

File: themes/scss/widgets/themes/light.scss (hunk 2 above)
Comment (xavijam) [inline themes/scss/widgets/themes/light.scss]: Yeah, I agree, I was on doubt.
```css
  .CDB-Chart-bar--standard {
    fill: $darkHighlighted;
  }
  .CDB-Chart-bar--timeSeries {
    fill: $highlightedAlternative;
  }
```
Sounds good?

Comment (viddo) [inline themes/scss/widgets/themes/light.scss]: :ok_woman:

File: themes/scss/widgets/themes/light.scss (hunk 3 above)
Comment (xavijam) [inline themes/scss/widgets/themes/light.scss]: :+1:

**Pull request merged**